Ruddarr/Views/Series/SeriesListItem.swift

```diff
@@ -0,0 +1,128 @@
+                    .font(ListItemHelper.secondaryTextStyle())
+                    .foregroundStyle(.secondary)
+                        
+                    HStack(spacing: ListItemHelper.listItemSpacing() / 2) {
+                        Image(systemName: "bookmark").symbolVariant(series.monitored ? .fill : .none).tag()
+                        Text(qualityProfileLabel).tag()
+                        if let sizeLabel = series.sizeLabel, !sizeLabel.isEmpty {
+                            Text(sizeLabel).tag()
+                        }
+                    }
+                    .lineLimit(1)
+                    .font(ListItemHelper.secondaryTextStyle())
+                    .foregroundStyle(.secondary)
+                }
+                .padding(.top, ListItemHelper.listItemSpacing())
+                .frame(maxHeight: .infinity, alignment: .top)
+            }
+            .padding(.top, ListItemHelper.listItemSpacing() / 3)
+            .padding(.bottom, ListItemHelper.listItemSpacing() / 2)
+            .padding(.horizontal, ListItemHelper.listItemSpacing())
+            .frame(height: ListItemHelper.posterHeight(), alignment: .top)
+        }
+        .frame(maxWidth: .infinity, alignment: .leading)
+        .background(.foreground.opacity(0.05))
+        .clipShape(RoundedRectangle(cornerRadius: ListItemHelper.posterRadius))
+    }
+
+    var poster: some View {
+        CachedAsyncImage(.poster, series.remotePoster, placeholder: series.title)
+            .clipShape(RoundedRectangle(cornerRadius: ListItemHelper.posterRadius))
+            .aspectRatio(2/3, contentMode: .fit)
+    }
+    
+    var qualityProfileLabel: String {
+        instance.qualityProfiles.first(where: { $0.id == series.qualityProfileId })?.name ?? String(localized: "Unknown")
+    }
+    
+    var nextAiringLabel: String {
+        guard let date = series.nextAiring else { return "" }
+        let calendar = Calendar.current
+        let time = date.formatted(date: .omitted, time: .shortened)
+        let weekday = date.formatted(.dateTime.weekday(.wide))
+
+        if calendar.isDateInToday(date) { return String(localized: "\(RelativeDate.today.label) at \(time)") }
+        if calendar.isDateInTomorrow(date) { return String(localized: "\(RelativeDate.tomorrow.label) at \(time)") }
+
+        guard let days = calendar.dateComponents([.day], from: Date.now, to: date).day else {
+            return ""
+        }
+
+        return days < 7
+            ? String(localized: "\(weekday) at \(time)")
+            : date.formatted(date: .abbreviated, time: .shortened)
+    }
+}
+
+#Preview {
+    let series: [Series] = PreviewData.load(name: "series")
+        .sorted { $0.year > $1.year }
+    
+    return ScrollView {
+        LazyVStack(spacing: ListItemHelper.listItemSpacing()) {
+            ForEach(series) { series in
+                SeriesGridItem(series: series)
+            }
+        }
+        .padding(.top, 0)
+        .viewPadding(.horizontal)
```

Comment:
Previously, the Mac app’s layout had no padding between the list and the search bar. This adds the necessary padding.